storybook/pages/StatusChatInputPage.qml

```diff
@@ -133,6 +125,9 @@ SplitView {
                 onRemovePaymentRequestPreview: (index) => {
                     d.paymentRequestModel.remove(index)
                 }
+                onOpenGifPopupRequest: {
```

Comment:
it would be nice to provide some exemplary handling of the request. In the simplest form we can just call the callback with some emoji.

Reply:
Or maybe even create some very small popup there, with e.g. two buttons rendering emojis. Click on button would trigger `cbOnGifSelected`, closing would trigger `cbOnClose`. This way we would have the details nicely presented there.

Reply:
Done!

https://github.com/user-attachments/assets/23bb699c-9177-4bdd-9919-7a02ee6464bf